x/posts/internal/types/post.go

```diff
@@ -260,6 +261,18 @@ func (p Post) ContentsEquals(other Post) bool {
 		ArePollDataEquals(p.PollData, other.PollData)
 }
 
+// GetPostHashtags returns all the post's hashtags without duplicates
+func (p Post) GetPostHashtags() []string {
+	hashtags := HashtagRegEx.FindAllString(p.Message, -1)
+	uniqueHashtags := commons.Unique(hashtags)
+	withoutHashtag := make([]string, len(uniqueHashtags))
+
+	for index, hashtag := range uniqueHashtags {
+		withoutHashtag[index] = strings.TrimLeft(hashtag, "#")
```

Comment:
Better to add an additional `TrimSpace` if the regex takes also those: 

```suggestion
		withoutHashtag[index] = strings.TrimSpace(strings.TrimLeft(hashtag, "#"))
```

Reply:
I don't know why but, with the suggested regEx, trim doesn't work and it don't remove the #

Reply:
> I don't know why but, with the suggested regEx, trim doesn't work and it don't remove the #

I think we first need to remove the spaces and then the `#` so it should be the opposite: 

```suggestion
		withoutHashtag[index] = strings.TrimLeft(strings.TrimSpace(hashtag), "#")
```